app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file_name, new_file_name = command.split()
+    if command != "mv":
+        return
+
+    if "/" in new_file_name:
```

Comment:
DIfferent OS use different slashes for separators. Use os.path.split to handle this